docs/05-PubsubForValidators.md

```diff
@@ -0,0 +1,58 @@
+# The x/pubsub Module for Validators
+
+Before Sommelier v7, the Steward Registry was used to record the CA and server endpoint of all steward processes. This database was then used by Strategy Providers (SPs) to send requests to each steward instance. 
+
+The Steward Registry has been deprecated in favor of a (mostly) generalized pubslisher/subscriber registry Cosmos SDK module, [x/pubsub](https://github.com/peggyjv/sommelier/tree/main/x/pubsub).
+
+For more information on x/pubsub itself, please refer to the [protocol documentation]()
+
+## Registering Steward as a Subscriber
+
+A `Subscriber` is an entity representing the identity of your steward instance. It contains your Sommelier address (either the validator or delegate address), server Certificate Authority, and the Steward server endpoint to which Strategy Providers (publishers) will send requests. 
```

Comment:
We should explicitly say that you should be using your orchestrator key for this. Steward is going to be using its Cosmos key as the Subscriber entity, and people should be using their orchestrator keys for that and definitely not their validator keys.